docs/reference-hw/lidars.md

```diff
@@ -0,0 +1,103 @@
+# LIDARs
+
+## **Velodyne 3D LIDAR Sensors**
+
+Velodyne Lidars which has ROS2 driver and tested by one or more community members are listed below:
+
+| Supported Products List | Range | FOV (V), (H)          | ROS2 Driver | Autoware Tested (Y/N) |
+| ----------------------- | ----- | --------------------- | ----------- | --------------------- |
+| Alpha Prime             | 245m  | (+15°)/(-25°), (360°) | Y           | Y                     |
+| Ultra Puck              | 200m  | (+15°)/(-25°), (360°) | Y           | Y                     |
+| Puck                    | 100m  | (+15°)/(-15°), (360°) | Y           | Y                     |
+| Puck Hi-res             | 100m  | (+10°)/(-10°), (360°) | Y           | Y                     |
+
+Link to ROS2 drivers:  
+ [https://github.com/ros-drivers/velodyne/tree/ros2/velodyne_pointcloud](https://github.com/ros-drivers/velodyne/tree/ros2/velodyne_pointcloud)  
+ [https://gitlab.com/autowarefoundation/autoware.auto/AutowareAuto/-/tree/master/src/drivers/velodyne_nodes](https://gitlab.com/autowarefoundation/autoware.auto/AutowareAuto/-/tree/master/src/drivers/velodyne_nodes)
```

Comment:
Maybe it's better to put AWF driver as well.
https://github.com/autowarefoundation/awf_velodyne/tree/tier4/universe